leftwm-core/src/state.rs

```diff
@@ -182,6 +182,9 @@ impl State {
                 new_window.set_states(old_window.states());
                 ordered.push(new_window.clone());
                 self.windows.remove(index);
+
+                let act = DisplayAction::SetWindowTags(new_window.handle, new_window.tags.clone());
+                self.actions.push_back(act);
```

Comment:
I think here would be some commentary helpful, why we want to `push_back` here exactly.